cypress/e2e/share-entities.cy.ts

```diff
@@ -103,9 +104,10 @@ describe('Share Entities', () => {
 
   it('should be able to edit and save', () => {
     cy.contains('h2', titleEntity4).click();
-    cy.get('aside.is-active').contains('button', 'Edit').click();
+    // cy.get('aside.is-active').contains('button', 'Edit').click();
+    clickOnEditEntity();
     cy.get('aside.is-active textarea').eq(0).clear();
-    cy.get('aside.is-active textarea').eq(0).type('Edited title');
+    cy.get('aside.is-active textarea').eq(0).type('Edited title', { force: true });
```

Comment:
I see a lot of `{force: true}`. Is something blocking the UI now?

Reply:
There is this wierd behavior by cypress, sometimes when typing in an input or a textarea, cypress does not type the whole text before continueing with the test. Using `{force: true}` seems to be a quick fix atleast for now, the internet seems to not have a clear way of fixing that problem